nodejs/awsx/apigateway/api.ts

```diff
@@ -445,6 +452,13 @@ interface LambdaAuthorizer {
     authorizerResultTtlInSeconds?: number;
 }
 
+interface CognitoAuthorizer {
+    type: "cognito_user_pools";
+    identitySource: string;
```

Comment:
intentional that identitySource is required here, but optional with LambdaAuthorizer?

Reply:
Yes per [docs](https://docs.aws.amazon.com/apigateway/api-reference/resource/authorizer/#identitySource)

The strange thing is that in some [examples](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-enable-cognito-user-pool.html) I dont see this set.

Reply:
Either way we do this for the user so they wont have to deal w it.